API/files/README.md

```diff
@@ -28,7 +28,8 @@ If no `content` is passed, then the path is treated as an empty directory
 ```js
 {
   path: '/tmp/myfile.txt',
-  node: DAGNode
+  hash: 'QmHash', // base58 encoded multihash
```

Comment:
got it. Right now, the content Ids are presented as base58 Encoded multihash, because there is no multibase nor multicodec. Once we have CID, the description will change to 'String representation of the CID', cause it is a self described Id :D